tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -16,12 +16,14 @@
 from pathlib import Path
```

Comment:
Thanks for doing x86 too! The commit message has a typo and probably should be labelled `test`

`test: x86_64: add host vs guest cpu feature test`

Reply:
fixed